worlds/timespinner/PyramidKeys.py

```diff
@@ -6,7 +6,7 @@ def get_pyramid_keys_unlock(world: MultiWorld, player: int) -> str:
     present_teleportation_gates: Tuple[str, ...] = (
         "GateKittyBoss",
         "GateLeftLibrary",
-        "GateMilitairyGate",
+        "GateMilitaryGate",

```

Comment:
We got to rename this back to GateMilitairyGate, as its part of contract between client and server, to fix this type we would first need to change the client to support both versions and then we can change the server side

Reply:
reverted